### PR TITLE
Add [FR] translation for basics/sigils 

### PR DESCRIPTION
**Pull request by ybycode**
In scope of issue #196.
@julienXX some more work for you, sorry :)

### CODE REVIEW

**Comment by doomspork** [comment]
Thank you @ybycode!

File: fr/lessons/basics/sigils.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+```elixir
+iex> ~w/i love elixir school/
+["i", "love", "elixir", "school"]
+
+iex> ~W/i love elixir school/
+["i", "love", "elixir", "school"]
+```
+
+On peut voir que ce qui est tapé entre les délimiteurs est séparée par des espaces et mis dans une liste. Pourtant, il n'y a pas de différence entre ces deux exemples. Encore une fois, la différence vient de l'interpolation et de l'échappement des séquences. Prenez l'exemple suivant:
```
Comment (julienXX) [inline fr/lessons/basics/sigils.md]: typo "séparée" -> "séparé"

**Comment by julienXX** [comment]
@ybycode looks good to me just a small typo :)

**Comment by ybycode** [comment]
@julienXX bien vu :+1: 
@doomspork it seems ready :)

**Comment by doomspork** [comment]
Thank you @ybycode and @julienXX 🎉

**Pull request merged**